src/scriptdev2/scripts/eastern_kingdoms/molten_core/boss_garr.cpp

```diff
@@ -110,43 +134,31 @@ struct mob_fireswornAI : public ScriptedAI
     mob_fireswornAI(Creature* pCreature) : ScriptedAI(pCreature)
     {
         m_pInstance = (ScriptedInstance*)pCreature->GetInstanceData();
-        Reset();
```

Comment:
I don't see any combat values that need to be initialized or reset here. The instance data is also no longer required and can be removed.